helix-term/src/ui/spinner.rs

```diff
@@ -32,7 +40,8 @@ pub struct Spinner {
 impl Spinner {
     /// Creates a new spinner with `frames` and `interval`.
     /// Expects the frames count and interval to be greater than 0.
-    pub fn new(frames: Vec<&'static str>, interval: u64) -> Self {
+    pub fn new(framestring: &str, interval: u64) -> Self {
+        let frames: Vec<char> = framestring.chars().collect();
```

Comment:
This will split a single character into two different ones given a certain input due to graphemes. Not sure how this is normally handled by helix.

Reply:
Again, I think `chars()` operates over unicode code points, not bytes. The braille characters appear to be two bytes long, in any case.